hardware/esp8266com/esp8266/boards.txt

```diff
@@ -57,6 +58,55 @@ generic.menu.FlashSize.4M=4M
 generic.menu.FlashSize.4M.build.flash_size=4M
 
 ##############################################################
+modwifi.name=Olimex MOD-WIFI-ESP8266(-DEV)
+
+modwifi.upload.tool=esptool
+modwifi.upload.speed=115200
+modwifi.upload.resetmethod=ck
+modwifi.upload.maximum_size=2097152
+modwifi.upload.wait_for_upload_port=true
+modwifi.serial.disableDTR=true
+modwifi.serial.disableRTS=true
+
+modwifi.build.mcu=esp8266
+modwifi.build.f_cpu=80000000L
+modwifi.build.board=MOD_WIFI_ESP8266
+modwifi.build.core=esp8266
+modwifi.build.variant=modwifi
```

Comment:
I suppose not. The role of each pin is printed in the silkscreen, so there's no reason to assign "D0", etc. Just use the GPIO numbers directly.